src/components/molecules/ScheduleEventListNG/ScheduleEventSubListNG.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useCallback, useMemo, useState } from "react";
+
+import { EVENTS_PREVIEW_LIST_LENGTH } from "settings";
+
+import { PersonalizedVenueEvent } from "types/venues";
+
+import { ScheduleItemNG } from "components/molecules/ScheduleItemNG";
+
+import "./ScheduleEventListNG.scss";
+
+export interface ScheduleEventSubListNGProps {
+  events: PersonalizedVenueEvent[];
+  title: string;
+}
+
+export const ScheduleEventSubListNG: React.FC<ScheduleEventSubListNGProps> = ({
+  events,
+  title,
+}) => {
+  const [showMoreEvents, setShowMoreEvents] = useState(false);
+  const toggleShowMoreEvents = useCallback(() => {
+    setShowMoreEvents(!showMoreEvents);
+  }, [showMoreEvents, setShowMoreEvents]);
```

Comment:
let's use `useShowHide` hook here instead.

Reply:
fixed in c7e4c1a90c7f5f57663972cbdd08ab5d051cb2d7